backend/spec/models/survey_spec.rb

```diff
@@ -5,4 +5,16 @@
     it { is_expected.to belong_to(:user).required }
     it { is_expected.to have_many(:questions) }
   end
+
+  describe '.by_user' do
```

Comment:
Also, need specs for the controller. They will be similar to these one, but simpler

Reply:
Ok! We will add those as well!